gwt-uibinder-processor/src/main/java/org/gwtproject/uibinder/processor/GwtResourceEntityResolver.java

```diff
@@ -30,7 +30,7 @@
           "http://dl.google.com/gwt/DTD/", "https://dl-ssl.google.com/gwt/DTD/"
       }).collect(toSet()));
 
-  private static final String RESOURCES = "org/gwtproject/uibinder/resources/";
+  private static final String RESOURCES = "org.gwtproject.uibinder.resources";
```

Comment:
I tested this out and you're correct.   Dangers of copying sources as-is and the empty catch block where it's used!   The APT filer uses the dotted notation whereas the older resource oracle used paths.